WordPress/Classes/Utility/In-App Feedback/AppRatingsUtility.swift

```diff
@@ -29,22 +29,9 @@ class AppRatingUtility: NSObject {
     private let defaults: UserDefaults
     private let featureFlagStore: RemoteFeatureFlagStore
     private var sections = [String: Section]()
-    private var promptingDisabledRemote: Bool {
-        RemoteFeatureFlag.inAppRating.enabled(using: featureFlagStore) == false
-    }
-
-    /// Don't prompt for reviews for internal builds
-    /// http://stackoverflow.com/questions/26081543/how-to-tell-at-runtime-whether-an-ios-app-is-running-through-a-testflight-beta-i?noredirect=1&lq=1
-    ///
-    private var promptingDisabledLocal: Bool = {
```

Comment:
Why was this logic there? And why are we removing it?

Reply:
That logic currently disables the prompt in beta builds. While I'm not certain why this is the case, enabling it shouldn't cause any harm. 

Besides, we need this feature enabled in beta builds for testing during the release rotation.

Reply:
On a 2nd thought, what do you think of enabling the prompt just for this release rotation then disable it afterwards?

While enabling this feature in beta builds and development isn't harmful, it could become bothersome, as it may lead to frequent and potentially disruptive prompts. This constant triggering could interfere with the testing and overall developer experience.

What do you think?

Reply:
I think we should have a way to enable it from the debug screen, to ease the testing. Also, an option to reset the count could be a useful debug tool. What do you think?

Reply:
> I think we should have a way to enable it from the debug screen, to ease the testing.

An easy way to achieve this is by adding a local feature flag, but I don't want to add another feature flag on top of the remote feature flag. I'll try to think of something else. 🧘

> Also, an option to reset the count could be a useful debug tool. What do you think?

This might be possible with the debug screen @justtwago is working on.

Reply:
@hassaanelgarem In beta builds, the debug screen is hidden, so any debug settings we add would need to be enabled programmatically prior to the beta release.

Is it alright if we address the debug feature in a separate PR? For this PR, I propose we:

- Revert the removal of the `promptingDisabledLocal` property.
- Force `promotingDisabledLocal` to always return false.
- Then, merge and release a new beta for testing.
- Meanwhile, we can work out the details of the debug feature.

Reply:
This plan sounds good to me!

Reply:
Resolved in b0bab7cc20f002023d16a843c4464449bc4d3c8e